packages/labextension/src/index.ts

```diff
@@ -39,7 +39,7 @@ async function activateCondaEnv(
   let tour: any;
   const { commands, shell } = app;
   const pluginNamespace = 'conda-env';
-  const command = 'jupyter_conda:open-ui';
+  const command = 'gator:open-ui';
```

Comment:
These command ids were left out of the renaming in case we wanted to publish another version, as they could have been seen as breaking changes for other extensions relying on these commands.

Reply:
Good catch, I'll undo this.